recipes/rte-rrtmgp/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Stop on any error
+set -e
+
+BUILD_DIR=build
+BUILD_TYPE=Debug
```

Comment:
Why debug?

Reply:
@isuruf This is legacy - the recipe reflects continuous integration practices in the Fortran repo. We could change to RelWithDebugInfo easily. Release build types don't pass accuracy tests on all platforms.